acceptance/.shared/kitchen_acceptance/.kitchen.vagrant.yml

```diff
@@ -38,8 +38,10 @@ fedora-21
 # freebsd-8
 # ubuntu-12.04
 # centos-7
-# centos-6
-
+  - name: centos-6
+    driver:
+      box: opscode-centos-6.6
+      box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_centos-6.6_chef-provisionerless.box
```

Comment:
Try:

```yaml
- name: centos-6
   driver:
      box: bento/centos-6.7
```

See this kitchen-vagrant change for more:
https://github.com/test-kitchen/kitchen-vagrant/commit/5697d9fd77617b41daf4b0f5165547449b856a43

If we would publish the CentOS Bento boxes to Atlas as `centos-MAJOR` you could drop even specifying a specific box name.

/cc @cheeseplus